Extensions/auto_tagger.js

```diff
@@ -297,16 +297,26 @@ XKit.extensions.auto_tagger = new Object({
 		}
 
 
-		if (XKit.extensions.auto_tagger.preferences.tag_source.value && obj.source_owner) {
-			var sourceTag;
-
-			if (XKit.extensions.auto_tagger.preferences.tag_person_replace_hyphens.value) {
-				sourceTag = XKit.extensions.auto_tagger.preferences.tag_source_prefix.value + obj.source_owner.replace(/-/g, ' ');
-			} else {
-				sourceTag = XKit.extensions.auto_tagger.preferences.tag_source_prefix.value + obj.source_owner;
+		if (XKit.extensions.auto_tagger.preferences.tag_source.value) {
+			var source;
+			if (obj.source_owner !== undefined) {
+				source = obj.source_owner;
+			}
+			else {
```

Comment:
from the linter: `Closing curly brace does not appear on the same line as the subsequent block`
```suggestion
			} else {
```